microsoftgraph/microsoft-graph-explorer-v4
# Fix: screen reader support and focus indication on links

## Overview

- Added aria-live tag on the status message, to enable the screen reader to read the message.

- Added an aria-label for the dismiss button on the status message and the terms of use message bar .

- Enabled multiline functionality on the terms of use message bar for better focus on the links when using keyboard navigation.

### Demo

![image](https://user-images.githubusercontent.com/18407044/86338554-22cb0e00-bc5b-11ea-989b-a0b0c4afc1be.png)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

To test the status message accessibility
Ensure a screen reader is activated (e.g Narrator)

Step 1: Launch the application on your browser.
Step 2: Navigate to “Run Query” buttons using tab key and activate it by pressing enter key.
Step 3: Observe screen reader is narrating “Status” message.

To test the focus indication on the links
Step 1: Launch the application on your browser.​
Step 2: Navigate to 'Microsoft APIs Terms of Use' and 'Microsoft Privacy Statement'  link by using tab key.
Step 3: Observe that a focus indicator is clearly visible.
